File: backend/components/metiers/administration_publique/utils/i18n/samples/sample_i18n_helper.test.py
# Test du helper i18n d’exemple (Python)
from .sample_i18n_helper import sample_i18n
def test_sample_i18n(capsys):
    assert sample_i18n('fr', 'Bonjour') is True
    captured = capsys.readouterr()
    assert '[I18N] [fr] Bonjour' in captured.out

File: backend/components/metiers/administration_publique/utils/i18n/samples/sample_i18n_helper.test.js
// Test du helper i18n d’exemple (JS)
const { sampleI18n } = require('./sample_i18n_helper');
describe('sampleI18n', () => {
  it('should log and return true', () => {
    expect(sampleI18n('fr', 'Bonjour')).toBe(true);
  });
});
